README.md

```diff
@@ -217,6 +217,65 @@ The `pages/api` directory is mapped to `/api/*`. Files in this directory are tre
 
 This project uses [`next/font`](https://nextjs.org/docs/basic-features/font-optimization) to automatically optimize and load Inter, a custom Google Font.
 
+## New Feature Add Process
+
+### Switch to main branch and pull
+
+```bash
+> git checkout main
+> git pull
+```
+
+### Create feature branch
+
+All feature work should be done on a branch. The branch should start with word feature. Dash (-) should be used as separator the branch name.
```

Comment:
Rephrase this sentence to just indicate how to name a feature branch. Provide guideline with examples.

Further, reader might get confused with "-" and "/".